WordPress/Classes/Models/Blog.m

```diff
@@ -458,6 +458,8 @@ - (BOOL)supports:(BlogFeature)feature
             return [self isHostedAtWPcom] && [self supportsSiteManagementServices];
         case BlogFeatureNoncePreviews:
             return [self supportsRestApi] && ![self isHostedAtWPcom];
+        case BlogFeatureMediaMetadataEditing:
+            return [self isHostedAtWPcom];
```

Comment:
@kurzee I've pushed up a change to address this. I've also disabled deletion for non-admins.